week-1/3-extra/2-piping.js

```diff
@@ -16,27 +16,35 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  let sum = num1 + num2;
+  let result = Math.round(sum * 10) / 10;
+  return result;
 }
+console.log(add(2.4, 5.3));
 
-function multiply() {
-
+function multiply(num1, num2) {
+  let result = num1 * num2;
+  return result;
 }
 
-function format() {
-
+function format(number) {
+  let result = `£${number}`;
+  return result;
 }
 
-const startingValue = 2
+const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode = 
-
+let badCode = add(startingValue, 10);
+badCode = multiply(badCode, 2);
+badCode = format(badCode);
```

Comment:
Were you unsure how to write this all on one line (this is what the instructions asked for on line 9)?

Reply:
Thanks Ben.  On the badCode task, I actually did/do not know how to write it on one line.  Now that we have completed week 3, wouldn't that be chaining? 
l

Reply:
Not quite, it's directly passing in the result of one function call into the other like so:
```javascript
format(multiply(add(startingValue, 10), 2))
```

You can't chain these functions as they return numbers and strings that don't have these functions attached to them.

Does that make sense?

Reply:
Yes, it makes sense now.  Thanks

Reply:
Is there something I need to do with this?
![image](https://user-images.githubusercontent.com/62774630/86845033-d0e21800-c0a0-11ea-82cb-2f218680350e.png)

Reply:
No, don't worry about the conflicts as this branch is not intended to be merged :)

Reply:
Thanks Ben